x/deployment/handler/handler.go

```diff
@@ -100,3 +102,26 @@ func handleMsgClose(ctx sdk.Context, keeper keeper.Keeper, mkeeper MarketKeeper,
 		Events: ctx.EventManager().Events(),
 	}, nil
 }
+
+func handleMsgCloseGroup(ctx sdk.Context, keeper keeper.Keeper, mkeeper MarketKeeper, msg types.MsgCloseGroup) (*sdk.Result, error) {
+	group, found := keeper.GetGroup(ctx, msg.ID)
+	if !found {
+		return nil, types.ErrGroupNotFound
+	}
+
+	// if Group already closed; return an error
+	if group.State == types.GroupClosed {
```

Comment:
Using ValidateOrderable breaks the simulations because it returns ErrGroupClosed for all states other than "open". Checking with the other calls to ValidateOrderable, I don't think it's safe to change behavior of the function and use it in the simulation operation check.

Reply:
_make_ a `group.ValidateCloseable()` ;)

Reply:
I created ValidateClosable().

To that point, should we just make all the fields of Group private? Add getters to the fields?

Reply:
Thanks.  We can't, unfortunately - private fields wouldn't be serialized.